test/pouch_cli_images_test.go

```diff
@@ -38,6 +39,20 @@ func (suite *PouchImagesSuite) TearDownTest(c *check.C) {
 	// TODO add cleanup work
 }
 
+// TestImagesQuietOption is to verify the quiet flag.
+func (suite *PouchImagesSuite) TestImagesQuietFlag(c *check.C) {
+	qOut, _, err := runCmd(exec.Command("pouch", "images", "-q"))
+	c.Assert(err, check.IsNil)
+
+	quietOut, _, err := runCmd(exec.Command("pouch", "images", "--quiet"))
+	c.Assert(err, check.IsNil)
+
+	c.Assert(qOut, check.Equals, quietOut)
+	if match, _ := regexp.MatchString("^[0-9a-f]+\n$", qOut); !match {
```

Comment:
I tried these code by using
```
Desktop $ cat main.go
package main

import (
	"regexp"
	"fmt"
)

func main(){
	qOut := "123456789"
	if match, _ := regexp.MatchString("^[0-9a-f]+\n$", qOut); !match {
 		fmt.Println("should return numeric ID, but got")
		fmt.Println(qOut)
	}
}

Desktop $ go run main.go
should return numeric ID, but got
123456789
```
I think 123456789 is a valid image ID, but the regexp will not pass.
Am I wrong?

Reply:
The ID is valid. 

But the cli uses `fmt.Println` so that each ID will have the `Carriage` char at the end. 
For the case, the qOut will contains the `\n`. That is why I put the `\n` before the `$` in the regrex.

Reply:
In this case, can we make pouchd pull an image in advance in the integration test of travisCI? 
@sunyuan3 @Letty5411

Reply:
BTW, the Windows uses `\r\n` as Carriage char. I don't have env to test that. 😢

Reply:
I am afraid yes. But we have not planned running on Windows.

Reply:
OK! 👍

Reply:
@allencloud Yes, I can pull an image in advance in the integration test of travisCI.